bin/generate_combined_qc_report.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# coding=utf-8
+
+# This file is part of MGnify genomes catalogue pipeline.
+#
+# MGnify genomes catalogue pipeline is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# MGnify genomes catalogue pipeline is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with MGnify genomes catalogue pipeline. If not, see <https://www.gnu.org/licenses/>.
+
+
+import argparse
+
+
+def main(qc, gunc, gtdb, outfile):
```

Comment:
Done in https://github.com/EBI-Metagenomics/genomes-catalogue-pipeline/commit/afa442768c2b274faccc35e66f6fe49e1d5d0f28 (different branch because the script already diverged)